src/containers/BooksForm.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { categories } from '../util/bookCategories';
+import {createBook} from '../actions/index';
+import { connect } from 'react-redux';
+import {PropTypes} from 'prop-types';
+
+class BooksForm extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            title: '',
+            category: ''
+        }
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
Hi, we can avoid this by using arrow functions. the know what `this` is. [OPTIONAL]

Reply:
Did it on purpose this way